WooCommerce/Classes/Analytics/WooAnalyticsEvent+AppIntents.swift

```diff
@@ -1,3 +1,5 @@
+import struct WooFoundation.WooAnalyticsEvent
```

Comment:
Seeing all these imports makes me feel like `WooAnalyticsEvent` should stay in WooCommerce, and instead implement a protocol from WooFoundation (`AnalyticsEvent`). The POS can then have `WooPOSAnalyticsEvent` which implements it too.

Reply:
`WooAnalyticsEvent` has been moved back to the main app as of the latest PR. As `Analytics.track(event: WooAnalyticsEvent)` is an extension of `Analytics`, I decided to not include this extension with an event protocol in this PR given the PR size. When we're ready to implement event tracking in the POS framework, we can make this protocol change as we see fit.